wordpress-mobile/WordPress-Android
# Feature/quick start tutorials

Part of #7707

This PR adds all Quick Start tutorials.
Originally, I wanted to make some sort of framework for the tutorials, but they are all quite different, spread all around the place and touch different parts of apps, so this turned out to be a bit problematic, and increased complexity without a clear return. Maybe this is something we can tackle in the future, but for now, I kept it simple and (hopefully) relatively organized, but everything still looks like a mess :(

Testing instructions:

1. Navigate to Quick Start activity from My Site fragment.
2. Click on each tutorial (create site is completed by default), follow it's instructions and make sure that they make sense, and after they are fulfilled the tutorial is marked as done (crossed out) in Quick Start activity.

Visually tutorials look like this:

**View site:**
![view_site](https://user-images.githubusercontent.com/728822/42201628-629b2712-7ea1-11e8-8ce3-db450b3c8fb4.png)

**Chose a theme:**
![themes](https://user-images.githubusercontent.com/728822/42201627-628866b8-7ea1-11e8-8a4b-20339f380b89.png)

**Customize site:**
![customize](https://user-images.githubusercontent.com/728822/42201623-62596aca-7ea1-11e8-8c17-17fc1d5682dd.png)

**Share:**
![sharing](https://user-images.githubusercontent.com/728822/42201625-6278855e-7ea1-11e8-9c3a-356351096e1b.png)
**Publish post:**
![create_post](https://user-images.githubusercontent.com/728822/42201622-6245b598-7ea1-11e8-8d1a-72e36782004e.png)

**Follow:**
![follow](https://user-images.githubusercontent.com/728822/42201624-6268c862-7ea1-11e8-9e2f-6fb99cb8614b.png)
